addons/ondevice-controls/src/GroupTabs.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
```

Comment:
I wonder where `GroupTabs` is used 🤔 . Searching the codebase for "GroupTabs" didn't seem to turn up anything. Tried deleting the file and didn't notice anything breaking. Do you know the purpose of the component?

Reply:
Ah, I wonder if it's related to what you mentioned here: https://github.com/storybookjs/react-native/pull/182#issuecomment-877669981

> Then future iterations on controls will include the following:
> * grouping like knobs had (maybe)

Reply:
Yeah you can see it here in the old knobs implementation.
https://github.com/storybookjs/react-native/blob/next-6.0/addons/ondevice-knobs/src/panel.js

Reply:
we should see if the current controls implementation on the web has groups